spec/06-constants.md

```diff
@@ -109,11 +109,16 @@ Constant Name | Description
 `PHP_INT_MAX` | `int`; the maximum representable value for an integer.
 `PHP_INT_MIN` | `int`; the minimum representable value for an integer.
 `PHP_INT_SIZE` |  `int`; the number of bytes used to represent an integer.
+`PHP_FLOAT_DIG` | `int`; the number of decimal digits that can be rounded into a float and back without precision loss.
+`PHP_FLOAT_EPSILON` | `float`; the smallest representable positive number `x`, so that `x + 1.0 != 1.0`.
+`PHP_FLOAT_MIN` | `int`; the smallest representable floating point number.
```

Comment:
More accurately, this is "the smallest representable normalized floating point number larger than zero". The current description sounds like PHP_FLOAT_MIN should be a large negative number, while it is a very small positive number (but not the smallest, because PHP_FLOAT_MIN excludes subnormals).

Reply:
I've updated the description to match yours. I also fixed the returned "type" had it set to int, when i should have been `float` according to the documentation.